cmd/babylond/cmd/root.go

```diff
@@ -241,6 +241,8 @@ func addModuleInitFlags(startCmd *cobra.Command) {
 
 	startCmd.Flags().String(flags.FlagKeyringBackend, flags.DefaultKeyringBackend, "Select keyring's backend (os|file|kwallet|pass|test)")
 	startCmd.Flags().String(flags.FlagChainID, "", "genesis file chain-id, if left blank will be randomly created")
+	startCmd.Flags().Bool(flagNoBlsPassword, false, "Generate BLS key without password protection (suitable for RPC nodes)")
```

Comment:
Would it improve UX if we set the default value to `true`? Then RPC node runners don't need to be aware of BLS stuff

Reply:
Yeah setting to true would be better.